aws/scripts/sh/website_installation.sh

```
@@ -1,5 +1,6 @@
 echo #### Install Software
-n install 21.7.1
+n install ${NODE_VERSION:?Node.js version is not set}
+n ${NODE_VERSION:?Node.js version is not set}
 git clone -b $WEBSITE_GIT_REPOSITORY_BRANCH "$WEBSITE_GIT_REPOSITORY_LINK.git" /codebuild-user/website
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)